readthedocs/proxito/middleware.py

```diff
@@ -185,34 +183,8 @@ def _set_request_attributes(self, request, unresolved_domain):
         Set attributes in the request from the unresolved domain.
 
         - Set ``request.unresolved_domain`` to the unresolved domain.
-        - If the domain needs to redirect, set the canonicalize attribute accordingly.
         """
         request.unresolved_domain = unresolved_domain
-        project = unresolved_domain.project
```

Comment:
Forgot to comment that now this method `_set_request_attributes` is just basically setting the unresolved domain in the request, it could be just an inline sentence, what do you think @humitos ?

Reply:
I think it's fine for now. It's pretty explicit (due its name).

Also, I remember you mentioned that you may want to include other attributes in the request and I said that could be everything in the same object. So, if we need to include multiple objects, this method will be helpful in the future as well.